client/chainlink.go

```diff
@@ -700,93 +703,53 @@ func (c *chainlink) do(
 
 // ConnectChainlinkNodes creates new chainlink clients
 func ConnectChainlinkNodes(e *environment.Environment) ([]Chainlink, error) {
-	return ConnectChainlinkNodesByCharts(e, []string{"chainlink"})
+	return ConnectChainlinkClientsFromEnv(e)
 }
 
+// ConnectChainlinkDBs creates new chainlink DBs clients
 func ConnectChainlinkDBs(e *environment.Environment) ([]*PostgresConnector, error) {
-	return ConnectChainlinkDBByCharts(e, []string{"chainlink"})
-}
-
-// ConnectChainlinkDBByCharts creates new chainlink DBs clients by charts
-func ConnectChainlinkDBByCharts(e *environment.Environment, charts []string) ([]*PostgresConnector, error) {
 	var dbs []*PostgresConnector
-	for _, chart := range charts {
-		pgUrls, err := e.Charts.Connections(chart).LocalURLsByPort("postgres", environment.HTTP)
+	dbURLs := e.URLs[chainlink2.DBsLocalURLsKey]
+	for _, u := range dbURLs {
```

Comment:
Could use some more descriptive variable names, not really sure what you're doing here, or why.

Reply:
Sure, fixed.